pages/landing.tsx

```diff
@@ -11,20 +11,16 @@ const Home: FC = () => {
     <Layout>
       <h1 className="mb-4">{t('header')}</h1>
       <h2 className="my-14">{t('description')}</h2>
-      <div className="grid grid-flow-row auto-rows-auto place-items-center text-xl gap-4">
-        <div className="text-center">
-          <ActionButton
-            href={t('intakeForm')}
-            style="primary"
-            text={t('withoutESRF')}
-          />
+      <div className="flex-container text-xl gap-4">
+        <div id="withoutESRF">
+          <Link href={t('intakeForm')} passHref>
```

Comment:
Link should probably be a part of the component, rather than wrapping LinkButton which is just an anchor tag